services/sts/systests/sts-itests/src/test/java/org/apache/cxf/systest/sts/itests/BasicSTSIntegrationTest.java

```diff
@@ -62,7 +63,7 @@ public Option[] getConfig() {
             when(!localRepository.isEmpty())
                 .useOptions(systemProperty("org.ops4j.pax.url.mvn.localRepository").value(localRepository)),
             //DO NOT COMMIT WITH THIS LINE ENABLED!!!
-            //KarafDistributionOption.keepRuntimeFolder(),
+            KarafDistributionOption.keepRuntimeFolder(),
```

Comment:
@rotty3000 I could confirm that despite being in configuration, the `org.ops4j.pax.url.mvn.localRepository` is not consulted when bundles are installed (as `mvn:`), may I ask you please to add the following setting (with addition to `org.ops4j.pax.url.mvn.localRepository`):
```
            when(!localRepository.isEmpty())
                .useOptions(systemProperty("org.ops4j.pax.url.mvn.defaultLocalRepoAsRemote").value("true")),

```

Reply:
:( no luck so far

Reply:
@rotty3000 may I ask you please for the permission to push to your branch? (a few experiments, don't want to drag you into that, thank you).

Reply:
@reta you should have it :)

![Screenshot from 2022-05-11 14-56-07](https://user-images.githubusercontent.com/146764/167925251-a808f57d-05bc-4b75-891d-0e96b81a876b.png)

Reply:
👍 TY!